src/resolvers/Query/catalogItems.js

```diff
@@ -12,15 +12,16 @@ import xformCatalogBooleanFilters from "../../utils/catalogBooleanFilters.js";
  * @summary Get a list of catalogItems
  * @param {Object} _ - unused
  * @param {ConnectionArgs} args - an object of all arguments that were sent by the client
+ * @param {String[]} [args.searchQuery] - limit to catalog items matching this text search query
  * @param {String[]} [args.shopIds] - limit to catalog items for these shops
- * @param {String[]} [args.tagIds] - limit to catalog items with this array of tags
+ * @param {String[]} [args.tagIds] - limit to catalog items with this array  of tags
```

Comment:
@loan-laux Just a quick removal of extra white space here.

Reply:
Done @Akarshit ✅